azurepipelines/e2e_test/scenarios/test_runner/apt_deployment.py

```diff
@@ -52,13 +52,12 @@ def test_AptDeployment(self):
         self.aduScenarioDefinition = DuScenarioDefinitionManager.FromOSEnvironment()
 
         test_device_id = self.aduScenarioDefinition.test_device_id
-        test_adu_group = self.aduScenarioDefinition.test_adu_group
+        test_adu_group = "gen1-test-device"
```

Comment:
Why did this needed to change?

Reply:
These are all vestigial at this point. They are being removed.